packages/server/computation_container/client/computation_to_db/client.cpp

```diff
@@ -217,7 +217,6 @@ std::string Client::writeTable(
 // Job を DB に新規登録する
 void Client::registerJob(const std::string &job_uuid, const int &status) const
 {
-    fs::create_directories(resultDbPath + job_uuid);
```

Comment:
I would prefer not to remove this code as it depends on the behavior of MC.

Reply:
[00effed](https://github.com/acompany-develop/QuickMPC/pull/150/commits/00effed174ce968ce3fa654612a380dc79a8600d)